glean_parser/kotlin.py

```diff
@@ -11,11 +11,15 @@
 from collections import OrderedDict
 import enum
 import json
+from pathlib import Path
+from typing import Any, Dict, List, Union  # noqa
 
+from . import metrics
+from . import pings
 from . import util
 
 
-def kotlin_datatypes_filter(value):
+def kotlin_datatypes_filter(value: util.JSONType) -> str:
```

Comment:
Oh, type annotations are ok in Python 3.5, but not variable annotations? Interesting

Reply:
Yeah -- the syntax for annotations on function arguments was added in 3.5, but annotation local variables was added in 3.6.  I think it was originally assumed that you would only want to type check on function boundaries, but it turns out that's not enough.  (The whole thing is awful in Python, honestly, but I do think it provides *some* value -- it detected a few type-correctness bugs found and fixed in this PR, for example).